packages/component-library-react/rollup.config.mjs

```diff
@@ -42,9 +42,8 @@ export default [
         globals: outputGlobals,
       },
     ],
-    external: [/@babel\/runtime/, 'react-dom', 'react'],
     plugins: [
-      peerDepsExternal({ includeDependencies: true }),
+      peerDepsExternal({ includeDependencies: false }),
```

Comment:
I propose removal of the `includeDependencies` option as it's no longer needed and it's confusing to have it specifically mentioned.